spec/models/playbook_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Thank you for creating this spec! There was one in minitest that needs to be deleted, not that it has been migrated to RSpec.
Please remove this file:
https://github.com/ChaelCodes/HuntersKeepers/blob/10e66ea18cf29db1e2e06b8a65ee2eb9ca63a967/test/models/playbook_test.rb#L15
Thank you for migrating the playbook spec to RSpec!

Reply:
Removed with commit 692f05e